pyproject.toml

```diff
@@ -41,7 +40,6 @@ classifiers = [
 ]
 
 [project.optional-dependencies]
-plugins = ["importlib-metadata;python_version<'3.8'"]
```

Comment:
I wonder if this should be kept as a dummy, empty `plugins` extra, so people can still do `pip install pygments[plugins]`?

And then remove it in the next major version?

Reply:
Re-added as a no-op.